metadata-models/src/main/pegasus/com/linkedin/dataJob/azkaban/AzkabanJobInfo.pdl

```diff
@@ -0,0 +1,12 @@
+namespace com.linkedin.dataJob.azkaban
+
+/**
+ * Metadata associated with a Azkaban job
+ */
+record AzkabanJobInfo includes AzkabanFlowInfo {
```

Comment:
I have a similar concern here. Rather than duplicating flow's own metadata by including it, can we have a field to capture the pointer for the flow like:
```
flow: AzkabanFlowUrn
```